client/src/components/Modal/Modal.js

```diff
@@ -49,24 +40,15 @@ const Modal = ({
     <>
       {modalOpen ? (
         <ModalContainer modalOpen={modalOpen}>
-          <ModalHeader>
-            <button onClick={() => setModalOpen(!modalOpen)}>✖️</button>
-            <form onSubmit={onSubmit}>
-              <input
-                type="text"
-                placeholder="Type a keyword"
-                value={value}
-                onChange={onChange}
-              />
-              <input type="submit" onClick={onChange} />
-            </form>
-          </ModalHeader>
+          {HeaderModal(setModalOpen, modalOpen, onSubmit, value, onChange)}
```

Comment:
mabye instead of giving the complete state into the component, you could create a function outside of it, and give this to the component , something like:

````js
const toogleModal = () => {
setModalOpen(!modalOpen)
}
`````
```suggestion
          {HeaderModal(toggleModal, onSubmit, value, onChange)}
```

Reply:
That is a good thought, but it blocks the input of the inputfield.

Reply:
Thanks for the good review! You can now see the deployment on heroku, after I fixed a problem in the server. 🎉